ntoskrnl/mm/freelist.c

```diff
@@ -21,7 +21,13 @@
 
 /* GLOBALS ****************************************************************/
 
+MM_DRIVER_VERIFIER_DATA LolData;
 PMMPFN MmPfnDatabase;
+#if DBG
+PKTHREAD MmPfnOwner;
+PCSTR MmPfnAcquiredAtFile;
+INT MmPfnAcquiredAtLine;
```

Comment:
Should you make use of them in your macros? They seem to be unused!

Reply:
Ha, nice catch, those come later, for the debug features mentioned in the description.